### ISSUE TITLE
.emacs.d/session.* files

### ISSUE THREAD

**Issue by DamienCassou** [tarsius/no-littering#38]
I regularly find `~/.emacs.d/session.<48 hex digits>` files. I don't know where they come from nor what they are used for but I guess they could be a good target for `no-littering`. I can send a PR if someone tells me the variable name.

**Comment by DamienCassou** [comment]
Apparently, these files are not controlled by a variable but by a function:

```emacs
(defun emacs-session-filename (session-id)
  "Construct a filename to save the session in based on SESSION-ID.
Return a filename in `user-emacs-directory', unless the session file
already exists in the home directory."
  (let ((basename (concat "session." session-id)))
    (locate-user-emacs-file basename
                            (concat ".emacs-" basename))))
```

Should `no-littering` advice around this function?

**Comment by tarsius** [comment]
Yeah, I guess we have to go down that road. Could you please create a pr? I would probably go with redefining the function altogether.

**Comment by tarsius** [comment]
Merged #39.
